dev/io.openliberty.io.smallrye.config/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -51,6 +51,8 @@ cleanup.use_this_for_non_static_method_access=false
 cleanup.use_this_for_non_static_method_access_only_if_necessary=true
 eclipse.preferences.version=1
 editor_save_participant_org.eclipse.jdt.ui.postsavelistener.cleanup=true
+formatter_profile=_config
```

Comment:
Did you want to change this?

Reply:
No, it looks like this was a result of updating the formatter settings with a newer version of eclipse (that part was deliberate). 

I don't think the `formatter_profile` name is at all relevant when we're using project-specific formatting settings but I'll take this file out since the new templates which have been added include `@author` tags which we don't use in openliberty.

Reply:
Apologies I was just trying to draw attention to the file in general and not any specific line.